sdks/android/Mentat/library/build.gradle

```diff
@@ -46,3 +49,50 @@ dependencies {
 repositories {
     mavenCentral()
 }
+
+// Publishing to jcenter/bintray.
+def libGroupId = properties.libGroupId
+def libRepoName = properties.libRepositoryName
+def libProjectName = properties.libProjectName
+def libProjectDescription = properties.libProjectDescription
+def libUrl = properties.libUrl
+def libVcsUrl = properties.libVcsUrl
+def libLicense = properties.libLicense
+
+Properties localProperties = null;
+if (project.rootProject.file('local.properties').canRead()) {
+    localProperties = new Properties()
+    localProperties.load(project.rootProject.file('local.properties').newDataInputStream())
+}
+
+version = rootProject.ext.library['version']
+
+task sourcesJar(type: Jar) {
+    from android.sourceSets.main.java.srcDirs
+    classifier = 'sources'
+}
+
+artifacts {
+    archives sourcesJar
+}
+
+group = libGroupId
+archivesBaseName = libProjectName
+
+bintray {
+    user = localProperties != null ? localProperties.getProperty("bintray.user") : ""
+    key = localProperties != null ? localProperties.getProperty("bintray.apikey") : ""
+
+    configurations = ['archives']
+    pkg {
+        repo = libRepoName
```

Comment:
Why do this extraction step (like `libFoo = properties.foo`) globally rather than in the `bintray` closure?

Reply:
No reason now, but at some point I've used these values multiple times. Will clean this up.

Reply:
Actually, there is a reason after all, something to do with groovy's scoping rules. I'll leave this as is.